exclusion_zones.civmap.json

```diff
@@ -89,6 +89,22 @@
 				[-1588, -2918]
 			]
 		},
+		{
+			"name": "Fort Sampaloc",
+			"owner": "Lambat",
```

Comment:
add a contact please

Reply:
still needs contact

Reply:
I deliberately didn't change this one. Lambat is an active democracy with a rapidly-changing government; I feel it's easier to put "Lambat" than to update this every time the government changes (frequently).

Reply:
can you link to a permanent discord invite then?